src/types/event-log.ts

```diff
@@ -1,13 +1,22 @@
+import type { Filter, SortOrder } from './message-types.js';
+
 export type Event = {
   watermark: string,
   messageCid: string
 };
 
-
 export type GetEventsOptions = {
   gt: string
 };
 
+
+export type FilteredQuery = {
+  filter: Filter;
```

Comment:
I've gotten confused by the ambiguity a few times in the past. With your permission, I'd like to propose that we should rename this to `AndFilter`, `Filters`, or `CompoundFilter`.

```suggestion
  andFilter: AndFilter;
```